tile_generator/bosh.py

```diff
@@ -153,8 +157,8 @@ def add_job(self, job):
 		)
 
 	def needs_zip(self, package):
-		# Only zip CF packages...
-		if not package.get('is_cf', False):
+		# Only zip package types that require single files
+		if not package.get('is_cf', False) and not package.get('zip_if_needed', False):
```

Comment:
I tried to get rid of as many `is_blah` flags as possible as it makes for a very poor re-usability. Could we do some inference here such as `if package_type == X` or `if release/job.endswith('.zip')`?